src/tests/ngraph_helpers/lpt_ngraph_functions/src/space_to_batch_function.cpp

```diff
@@ -0,0 +1,65 @@
+                                                            const FakeQuantizeOnData& fq_on_data,
+                                                            const std::vector<size_t>& block_shape,
+                                                            const std::vector<size_t>& pads_begin,
+                                                            const std::vector<size_t>& pads_end) {
+    const auto input = std::make_shared<ngraph::opset1::Parameter>(input_type, input_shape);
+
+    std::shared_ptr<Node> parent = fq_on_data.empty() ?
+        std::dynamic_pointer_cast<ngraph::Node>(input) :
+        makeFakeQuantize(input, input_type, fq_on_data);
+
+    parent = std::make_shared<ngraph::opset2::SpaceToBatch>(
+        parent,
+        std::make_shared<ngraph::opset1::Constant>(ngraph::element::i64, ngraph::Shape{ block_shape.size() }, block_shape),
+        std::make_shared<ngraph::opset1::Constant>(ngraph::element::i64, ngraph::Shape{ pads_begin.size() }, pads_begin),
+        std::make_shared<ngraph::opset1::Constant>(ngraph::element::i64, ngraph::Shape{ pads_end.size() }, pads_end));
+
+    ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(parent)};
+    return std::make_shared<ngraph::Function>(results, ngraph::ParameterVector{ input }, "SpaceToBatchFunction");
+}
+
+std::shared_ptr<ngraph::Function> SpaceToBatchFunction::get(const ngraph::PartialShape& input_shape,
+                                                            const ngraph::element::Type input_type,
+                                                            const ngraph::builder::subgraph::DequantizationOperations& dequantization_before,
+                                                            const std::vector<size_t>& block_shape,
+                                                            const std::vector<size_t>& pads_begin,
+                                                            const std::vector<size_t>& pads_end,
+                                                            const ngraph::builder::subgraph::DequantizationOperations& dequantization_after) {
+    const auto input = std::make_shared<ngraph::opset1::Parameter>(input_type, input_shape);
+
+    std::shared_ptr<Node> parent = dequantization_before.empty() ?
+        std::dynamic_pointer_cast<ngraph::Node>(input) :
+        makeDequantization(input, dequantization_before);
+
+    parent = std::make_shared<ngraph::opset2::SpaceToBatch>(
+        parent,
+        std::make_shared<ngraph::opset1::Constant>(ngraph::element::i64, ngraph::Shape{ block_shape.size() }, block_shape),
+        std::make_shared<ngraph::opset1::Constant>(ngraph::element::i64, ngraph::Shape{ pads_begin.size() }, pads_begin),
+        std::make_shared<ngraph::opset1::Constant>(ngraph::element::i64, ngraph::Shape{ pads_end.size() }, pads_end));
+
+    if (!dequantization_after.empty()) {
+        parent = makeDequantization(parent, dequantization_after);
+    }
```

Comment:
Minor: we can not check `dequantization_after` because `makeDequantization` just returns parent if `dequantization_after` is empty

Reply:
done, thanks